### PR TITLE
install podman instead of docker for centoses and fedora

### PR DESCRIPTION
**Pull request by zmiklank**
Resolves: https://github.com/sclorg/sclorg-testing-farm/issues/35

### CODE REVIEW

File: plans/centos7.fmf [hunk 1]
```diff
@@ -15,7 +15,7 @@ prepare:
     script: |
         # TODO install packages by ansible playbook
         curl -o /etc/yum.repos.d/distgen-epel-7.repo https://copr.fedorainfracloud.org/coprs/praiskup/distgen/repo/epel-7/praiskup-distgen-epel-7.repo
-        yum -y install docker perl git centos-release-scl-rh rsync groff-base centos-release-openshift-origin epel-release http://cbs.centos.org/kojifiles/packages/golang-github-cpuguy83-go-md2man/1.0.4/4.0.el7/x86_64/golang-github-cpuguy83-go-md2man-1.0.4-4.0.el7.x86_64.rpm
+        yum -y install podman podman-docker perl git centos-release-scl-rh rsync groff-base centos-release-openshift-origin epel-release http://cbs.centos.org/kojifiles/packages/golang-github-cpuguy83-go-md2man/1.0.4/4.0.el7/x86_64/golang-github-cpuguy83-go-md2man-1.0.4-4.0.el7.x86_64.rpm
```
Comment (phracek) [inline plans/centos7.fmf:18]: CentOS7 and RHEL7 use docker. Not podman.

Comment (zmiklank) [reply plans/centos7.fmf:18]: Isn't it only RHEL7? Honza says in the linked issue this:
"yum install docker results in installing moby aka open-source docker. That's likely not what we want, podman should be installed everywhere, except RHEL-7."

Comment (phracek) [reply plans/centos7.fmf:18]: Docker and podman are available here: http://mirror.centos.org/centos/7/extras/x86_64/Packages/.
The latest package http://mirror.centos.org/centos/7/extras/x86_64/Packages/docker-1.13.1-209.git7d71120.el7.centos.x86_64.rpm
contains dockerd and binaries as well.
```bash
$ ls -laR usr
total 0
drwxr-xr-x   6 phracek  staff   192 Nov 10 12:37 .
drwx------@ 65 phracek  staff  2080 Nov 10 12:38 ..
drwxr-xr-x   7 phracek  staff   224 Nov 10 12:37 bin
drwxr-xr-x   4 phracek  staff   128 Nov 10 12:37 lib
drwxr-xr-x   3 phracek  staff    96 Nov 10 12:37 libexec
drwxr-xr-x   9 phracek  staff   288 Nov 10 12:37 share

usr/bin:
total 109024
drwxr-xr-x  7 phracek  staff       224 Nov 10 12:37 .
drwxr-xr-x  6 phracek  staff       192 Nov 10 12:37 ..
-rwxr-xr-x@ 1 phracek  staff  10807432 Mar  2  2022 docker-containerd-current
-rwxr-xr-x@ 1 phracek  staff   2585896 Mar  2  2022 docker-containerd-shim-current
-rwxr-xr-x@ 1 phracek  staff  10015424 Mar  2  2022 docker-ctr-current
lrwxr-xr-x@ 1 phracek  staff        32 Mar  2  2022 docker-storage-setup -> /usr/bin/container-storage-setup
-rwxr-xr-x@ 1 phracek  staff  32400288 Mar  2  2022 dockerd-current
```

And docker-client http://mirror.centos.org/centos/7/extras/x86_64/Packages/docker-client-1.13.1-209.git7d71120.el7.centos.x86_64.rpm contains

```bash
$ ls -laR usr
total 0
drwxr-xr-x   4 phracek  staff   128 Nov 10 12:42 .
drwx------@ 66 phracek  staff  2112 Nov 10 12:42 ..
drwxr-xr-x   3 phracek  staff    96 Nov 10 12:42 bin
drwxr-xr-x   3 phracek  staff    96 Nov 10 12:42 share

usr/bin:
total 25744
drwxr-xr-x  3 phracek  staff        96 Nov 10 12:42 .
drwxr-xr-x  4 phracek  staff       128 Nov 10 12:42 ..
-rwxr-xr-x@ 1 phracek  staff  13177008 Mar  2  2022 docker-current
```
```

Comment (zmiklank) [reply plans/centos7.fmf:18]: Fixed.

**Review by phracek — APPROVED** [review]

**Pull request merged**